component/azstorage/block_blob.go

```diff
@@ -582,11 +583,21 @@ func (bb *BlockBlob) List(prefix string, marker *string, count int32) ([]*intern
 				return blobList, nil, err
 			}
 		} else {
+			var mode os.FileMode
+			if blobInfo.Properties.Permissions != nil {
```

Comment:
Should we add a nil check for blobInfo.Properties? Is there a possibility of it to be nil?

Reply:
For FNS accounts this is nil